### DISCUSSION TITLE
OriginLab Pro

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#9] [General]
# Origin

A community-driven list of software (apps and games) that work, are emulated or don't work on Windows on ARM

https://windowsonarm.org/

**Comment by OpenSource03** [comment]
I classified it as "Emulated", but I can not run it unfortunately as it requires a license. I would appreciate if the author or someone else would be able to let us know if it works on their ARM machine.

**Reply by tmdghks5229** [reply]
It works well with prism and the Icon is wrong

**Reply by OpenSource03** [reply]
Thanks for confirming, and yes, indeed, the icon seem to be wrong, haha. I will update it shortly.

**Reply by OpenSource03** [reply]
Updated.
